incubator/datadog-apm/ci/test-values.yaml

```
@@ -9,22 +9,20 @@ datadog:
   apiKeyExistingSecret:   # <DATADOG_API_KEY_SECRET>
   appKey: YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY  # <DATADOG_APP_KEY>
   appKeySecretName:   # <DATADOG_APP_KEY_SECRET>
-  clusterChecks:
-    enabled: false
   logLevel: INFO
   site: datadoghq.com
 
 clusterAgent:
   image:
     repository: datadog/agent
     pullPolicy: Always
-    # Overrides the image tag which defaults to the chart appVersion, created by datadog
-    tag: 7.23.0
+    # Overrides the image tag whose default is {{ .Chart.AppVersion }}
```

Using this format will put the comment into the README docs. It's not required since this is incubator, but nice to have if we ever want to promote to stable.

```suggestion
    # -- Overrides the image tag whose default is {{ .Chart.AppVersion }}
```